### PR TITLE
Adding datasources for alert, dashboard, derived_metric, events

### PR DESCRIPTION
**Pull request by dkuldeep22**
This changelist contains changes to add datasources for following wavefront components.
1. Alert/Alerts
2. Event/Events
3. DerivedMetric/DerivedMetrics
4. Dashboard/Dashboards

### CODE REVIEW

**Review by yogeshprasad — APPROVED** [review]

File: docs/data-sources/derived_metrics.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+* `derived_metrics` - List of all derived metrics in Wavefront.
+    * `name` - The name of the Derived Metric in Wavefront.
+    * `id` - The id of the Derived Metric in Wavefront.
+    * `query`- A Wavefront query that is evaluated at regular intervals (default is 1 minute).
+    * `minutes` - How frequently the query generating the derived metric is run.
+    * `in_trash` - A Boolean variable indicating trash status.
+    * `tags` - A set of tags assigned to the Derived Metric.
+    * `query_failing` - A Boolean variable indicating whether query is failing for derived metric.
+    * `last_error_message` - Last error message occurred.
+    * `last_failed_time` - Timestamp of last failed derived metric.
```
Comment (Margarita-Staneva) [inline docs/data-sources/derived_metrics.md:31]: Timestamp of the last failed derived metric.

File: docs/data-sources/derived_metrics.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+## Attribute Reference
+
+* `derived_metrics` - List of all derived metrics in Wavefront.
+    * `name` - The name of the Derived Metric in Wavefront.
+    * `id` - The id of the Derived Metric in Wavefront.
+    * `query`- A Wavefront query that is evaluated at regular intervals (default is 1 minute).
+    * `minutes` - How frequently the query generating the derived metric is run.
+    * `in_trash` - A Boolean variable indicating trash status.
+    * `tags` - A set of tags assigned to the Derived Metric.
+    * `query_failing` - A Boolean variable indicating whether query is failing for derived metric.
```
Comment (Margarita-Staneva) [inline docs/data-sources/derived_metrics.md:29]: ... failing for the derived metric.

File: docs/data-sources/derived_metrics.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+## Attribute Reference
+
+* `derived_metrics` - List of all derived metrics in Wavefront.
+    * `name` - The name of the Derived Metric in Wavefront.
+    * `id` - The id of the Derived Metric in Wavefront.
+    * `query`- A Wavefront query that is evaluated at regular intervals (default is 1 minute).
+    * `minutes` - How frequently the query generating the derived metric is run.
+    * `in_trash` - A Boolean variable indicating trash status.
+    * `tags` - A set of tags assigned to the Derived Metric.
```
Comment (Margarita-Staneva) [inline docs/data-sources/derived_metrics.md:28]: A set of tags assigned to the derived metric.

File: docs/data-sources/derived_metrics.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    * `query_failing` - A Boolean variable indicating whether query is failing for derived metric.
+    * `last_error_message` - Last error message occurred.
+    * `last_failed_time` - Timestamp of last failed derived metric.
+    * `additional_information` - User-supplied additional explanatory information for the derived metric.
```
Comment (Margarita-Staneva) [inline docs/data-sources/derived_metrics.md:32]: User-supplied additional explanatory information about the derived metric.

File: docs/data-sources/derived_metrics.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    * `create_user_id` - The id of user who created derived metric.
```
Comment (Margarita-Staneva) [inline docs/data-sources/derived_metrics.md:33]: The ID of the user who created the derived metric.

File: docs/data-sources/derived_metrics.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+    * `update_user_id` - The id of user who updated derived metric.
```
Comment (Margarita-Staneva) [inline docs/data-sources/derived_metrics.md:34]: The ID of the user who updated the derived metric.

File: docs/data-sources/derived_metrics.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+    * `status` - The status of derived metric.
```
Comment (Margarita-Staneva) [inline docs/data-sources/derived_metrics.md:35]: The status of the derived metric.

File: docs/data-sources/derived_metrics.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+    * `hosts_used` - A list of hosts used in derived metric.
```
Comment (Margarita-Staneva) [inline docs/data-sources/derived_metrics.md:36]: A list of hosts used in the derived metric.

File: docs/data-sources/derived_metrics.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+    * `last_processed_millis` - The last processed timestamp.
+    * `process_rate_minutes` -  The specified query is executed every `process_rate_minutes` minutes.
+    * `points_scanned_at_last_query` - The number of points scanned when last query was executed.
```
Comment (Margarita-Staneva) [inline docs/data-sources/derived_metrics.md:39]: The number of points scanned when the last query was executed.

File: docs/data-sources/derived_metrics.md [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+    * `include_obsolete_metrics` - A boolean flag indicating whether to include obsolete metrics or not.
```
Comment (Margarita-Staneva) [inline docs/data-sources/derived_metrics.md:40]: A Boolean...

File: docs/data-sources/derived_metrics.md [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+    * `last_query_time` - The timestamp when query was executed last time.
```
Comment (Margarita-Staneva) [inline docs/data-sources/derived_metrics.md:41]: The timestamp indicating the last time the query was executed.

File: docs/data-sources/derived_metrics.md [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+    * `metrics_used` - A list of metrics used in a derived metric.
```
Comment (Margarita-Staneva) [inline docs/data-sources/derived_metrics.md:42]: A list of metrics used in the derived metric.

File: docs/data-sources/derived_metrics.md [hunk 13]
```diff
@@ -0,0 +1,49 @@
...
+    * `query_qb_enabled` - A boolean flag for enabling `query_qb`
```
Comment (Margarita-Staneva) [inline docs/data-sources/derived_metrics.md:43]: A Boolean flag for enabling `query_qb`.

File: docs/data-sources/derived_metrics.md [hunk 14]
```diff
@@ -0,0 +1,49 @@
...
+    * `deleted` - A Boolean flag indicating derived metric deleted or not.
```
Comment (Margarita-Staneva) [inline docs/data-sources/derived_metrics.md:44]: A Boolean flag indicating whether the derived metric is deleted or not.

File: docs/data-sources/derived_metrics.md [hunk 15]
```diff
@@ -0,0 +1,49 @@
...
+    * `created_epoch_millis` - The timestamp in epoch millis when derived metrics is created.
```
Comment (Margarita-Staneva) [inline docs/data-sources/derived_metrics.md:45]: The timestamp in epoch milliseconds indicating when the derived metric is created.

File: docs/data-sources/derived_metrics.md [hunk 16]
```diff
@@ -0,0 +1,49 @@
...
+    * `updated_epoch_millis` - The timestamp in epoch millis when derived metrics is updated.
```
Comment (Margarita-Staneva) [inline docs/data-sources/derived_metrics.md:46]: The timestamp in epoch milliseconds indicating when the derived metric is updated.

File: docs/data-sources/event.md [hunk 17]
```diff
@@ -0,0 +1,35 @@
+---
+layout: "wavefront"
+page_title: "Wavefront: Event"
+description: |-
+Get the information for a given event from Wavefront
```
Comment (Margarita-Staneva) [inline docs/data-sources/event.md:5]: Get the information about a certain Wavefront event.

File: docs/data-sources/event.md [hunk 18]
```diff
@@ -0,0 +1,35 @@
...
+---
+
+# Data Source: wavefront_event
+
+Use this data source to get information for a given event from Wavefront.
```
Comment (Margarita-Staneva) [inline docs/data-sources/event.md:10]: Use this data source to get information about a certain Wavefront event.

File: docs/data-sources/event.md [hunk 19]
```diff
@@ -0,0 +1,35 @@
...
+
+## Argument Reference
+* `id` - (Required) The id associated with the event data to be fetched.
```
Comment (Margarita-Staneva) [inline docs/data-sources/event.md:13]: The ID...

File: docs/data-sources/event.md [hunk 20]
```diff
@@ -0,0 +1,35 @@
...
+
+## Example Usage
+
+```hcl
+# Get the info for all derived metrics
```
Comment (Margarita-Staneva) [inline docs/data-sources/event.md:18]: Get the information about a Wavefront event by its ID.

File: docs/data-sources/event.md [hunk 21]
```diff
@@ -0,0 +1,35 @@
...
+data "wavefront_event" "example" {
+   id = "sample-event-id"
+}
+```
+
+## Attribute Reference
+
+* `name` - The name of the Event in Wavefront.
```
Comment (Margarita-Staneva) [inline docs/data-sources/event.md:26]: event should be lowercase.

File: docs/data-sources/event.md [hunk 22]
```diff
@@ -0,0 +1,35 @@
...
+* `id` - The id of the Event in Wavefront.
```
Comment (Margarita-Staneva) [inline docs/data-sources/event.md:27]: The ID of the event in Wavefront.

File: docs/data-sources/event.md [hunk 23]
```diff
@@ -0,0 +1,35 @@
...
+* `start_time`- The start time of the Event in epoch milliseconds.
```
Comment (Margarita-Staneva) [inline docs/data-sources/event.md:28]: ...event...

File: docs/data-sources/event.md [hunk 24]
```diff
@@ -0,0 +1,35 @@
...
+* `end_time` - The end time of the Event in epoch milliseconds.
```
Comment (Margarita-Staneva) [inline docs/data-sources/event.md:29]: ...event...

File: docs/data-sources/event.md [hunk 25]
```diff
@@ -0,0 +1,35 @@
...
+* `severit` - The severity category of the Event.
```
Comment (Margarita-Staneva) [inline docs/data-sources/event.md:30]: ...event...

IMHO there's typo in the attribute. It should be `severity`

File: docs/data-sources/event.md [hunk 26]
```diff
@@ -0,0 +1,35 @@
...
+* `type` - The type of Event.
```
Comment (Margarita-Staneva) [inline docs/data-sources/event.md:31]: The type of the event.

File: docs/data-sources/event.md [hunk 27]
```diff
@@ -0,0 +1,35 @@
...
+* `details` - The description about the Event.
```
Comment (Margarita-Staneva) [inline docs/data-sources/event.md:32]: The description of the event.

File: docs/data-sources/event.md [hunk 28]
```diff
@@ -0,0 +1,35 @@
...
+* `is_ephemeral` - A boolean flag, if true, creates a point-in-time Event( i.e. with no duration)
```
Comment (Margarita-Staneva) [inline docs/data-sources/event.md:33]: A Boolean flag. If set to `true`, creates a point-in-time event( i.e. with no duration).

File: docs/data-sources/event.md [hunk 29]
```diff
@@ -0,0 +1,35 @@
...
+* `annotations` - Annotations associated with the Event.
```
Comment (Margarita-Staneva) [inline docs/data-sources/event.md:34]: ...event.

File: docs/data-sources/event.md [hunk 30]
```diff
@@ -0,0 +1,35 @@
...
+* `tags` - A set of tags assigned to the Event.
```
Comment (Margarita-Staneva) [inline docs/data-sources/event.md:35]: ...event.

File: docs/data-sources/events.md [hunk 31]
```diff
@@ -0,0 +1,32 @@
+---
+layout: "wavefront"
+page_title: "Wavefront: Events"
+description: |-
+Get the information for all events from Wavefront
+---
+
+# Data Source: wavefront_events
+
+Use this data source to get information for all events from Wavefront.
```
Comment (Margarita-Staneva) [inline docs/data-sources/events.md:10]: Use this data source to get information about all Wavefront events.

File: docs/data-sources/events.md [hunk 32]
```diff
@@ -0,0 +1,32 @@
+---
+layout: "wavefront"
+page_title: "Wavefront: Events"
+description: |-
+Get the information for all events from Wavefront
```
Comment (Margarita-Staneva) [inline docs/data-sources/events.md:5]: Get the information about all Wavefront events.

File: docs/data-sources/events.md [hunk 33]
```diff
@@ -0,0 +1,32 @@
...
+---
+
+# Data Source: wavefront_events
+
+Use this data source to get information for all events from Wavefront.
+
+## Example Usage
+
+```hcl
+# Get the info for all derived metrics
```
Comment (Margarita-Staneva) [inline docs/data-sources/events.md:15]: This is incorrect, please fix it.

File: docs/data-sources/events.md [hunk 34]
```diff
@@ -0,0 +1,32 @@
...
+data "wavefront_events" "example" {
+}
+```
+
+## Attribute Reference
+
+* `events` - List of all events in Wavefront.
```
Comment (Margarita-Staneva) [inline docs/data-sources/events.md:22]: List of all events in Wavefront. For each event you will see a list of attributes.

File: docs/data-sources/events.md [hunk 35]
```diff
@@ -0,0 +1,32 @@
...
+    * `name` - The name of the Event in Wavefront.
```
Comment (Margarita-Staneva) [inline docs/data-sources/events.md:23]: Same comments from the review of event.md apply here.

File: docs/data-sources/events.md [hunk 36]
```diff
@@ -0,0 +1,32 @@
...
+    * `id` - The id of the Derived Metric in Wavefront.
```
Comment (Margarita-Staneva) [inline docs/data-sources/events.md:24]: The ID of the event in Wavefront.

File: docs/resources/event.md [hunk 37]
```diff
@@ -0,0 +1,61 @@
+---
+layout: "wavefront"
+page_title: "Wavefront: Event"
+description: |-
+Provides a Wavefront Event resource.  This allows events to be created, updated, and deleted.
```
Comment (Margarita-Staneva) [inline docs/resources/event.md:5]: remove the extra space between sentences

File: docs/resources/event.md [hunk 38]
```diff
@@ -0,0 +1,61 @@
...
+---
+
+# Resource : wavefront_event
+
+Provides a Wavefront Event resource.  This allows events to be created, updated, and deleted.
```
Comment (Margarita-Staneva) [inline docs/resources/event.md:10]: Provides a Wavefront event resource. 

+ remove the extra space between sentences

File: docs/resources/event.md [hunk 39]
```diff
@@ -0,0 +1,61 @@
...
+
+## Example usage
```
Comment (Margarita-Staneva) [inline docs/resources/event.md:12]: Usage

File: docs/resources/event.md [hunk 40]
```diff
@@ -0,0 +1,61 @@
...
+
+```hcl
+resource "wavefront_event" "evente" {
```
Comment (Margarita-Staneva) [inline docs/resources/event.md:15]: "evente"?

File: docs/resources/event.md [hunk 41]
```diff
@@ -0,0 +1,61 @@
...
+    "eventTag1"
+  ]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the alert as it is displayed in Wavefront.
```
Comment (Margarita-Staneva) [inline docs/resources/event.md:32]: Do you mean event resource, not alert?

File: docs/resources/event.md [hunk 42]
```diff
@@ -0,0 +1,61 @@
...
+* `tags` - (Optional) A set of tags to assign to this resource.
+* `annotations` - (Required) The annotations associated with the Event.
```
Comment (Margarita-Staneva) [inline docs/resources/event.md:34]: Lowercase event. Same applies to the arguments below.

File: docs/resources/event.md [hunk 43]
```diff
@@ -0,0 +1,61 @@
...
+    "eventTag1"
+  ]
+  start_time = 1490000000000
+  end_time = 1490000000001
+}
+```
+
+## Import
+
+Events can be imported using the `id`, e.g.
```
Comment (Margarita-Staneva) [inline docs/resources/event.md:57]: You can import events by using the `id`, for example:

**Review by Margarita-Staneva — CHANGES_REQUESTED** [review]
Please make the necessary changes before merging. If you have any question about my comments, let me know.

File: docs/data-sources/alert.md [hunk 44]
```diff
@@ -0,0 +1,47 @@
...
+# Get the information about the alert.
+data "wavefront_alert" "example" {
+  id = "alert-id"
+}
+```
+
+## Attribute Reference
+
+* `name` - The name of the alert as it is displayed in Wavefront.
+* `id` - The ID of the derived metric in Wavefront.
```
Comment (Margarita-Staneva) [inline docs/data-sources/alert.md:27]: Shouldn't this be the ID of the alert?

File: docs/data-sources/alert.md [hunk 45]
```diff
@@ -0,0 +1,47 @@
...
+* `alert_type`- The type of alert in Wavefront.
+* `additional_information` - User-supplied additional explanatory information about this alert.
+* `target` - A email address or integration endpoint (such as PagerDuty or webhook) to notify when the alert status changes.
```
Comment (Margarita-Staneva) [inline docs/data-sources/alert.md:30]: An email address or integration endpoint (such as PagerDuty or webhook) to notify when the alert status changes.

File: docs/data-sources/alerts.md [hunk 46]
```diff
@@ -0,0 +1,44 @@
...
+```
+
+## Attribute Reference
+
+* `alerts` - List of all alerts in Wavefront. For each alert you will see a list of attributes.
+  * `name` - The name of the alert as it is displayed in Wavefront.
+  * `id` - The ID of the alert in Wavefront.
+  * `alert_type`- The type of alert in Wavefront.
+  * `additional_information` - User-supplied additional explanatory information about this alert.
+  * `target` - A email address or integration endpoint (such as PagerDuty or webhook) to notify when the alert status changes.
```
Comment (Margarita-Staneva) [inline docs/data-sources/alerts.md:27]: An email address or integration endpoint (such as PagerDuty or webhook) to notify when the alert status changes.

File: docs/data-sources/derived_metric.md [hunk 47]
```diff
@@ -0,0 +1,52 @@
...
+description: |-
+Get the information about a certain Wavefront derived metric by its ID.
+---
+
+# Data Source: wavefront_derived_metrics
+
+Use this data source to get information about a certain Wavefront derived metric by its ID.
+
+## Argument Reference
+* `id` - (Required) The id associated with the derived metric data to be fetched.
```
Comment (Margarita-Staneva) [inline docs/data-sources/derived_metric.md:13]: (Required) The ID...

File: docs/data-sources/event.md [hunk 48]
```diff
@@ -0,0 +1,35 @@
...
+## Attribute Reference
+
+* `name` - The name of the event in Wavefront.
+* `id` - The ID of the event in Wavefront.
+* `start_time`- The start time of the event in epoch milliseconds.
+* `end_time` - The end time of the event in epoch milliseconds.
+* `severity` - The severity category of the event.
+* `type` - The type of the event.
+* `details` - The description of the event.
+* `is_ephemeral` - A Boolean flag. If set to `true`, creates a point-in-time event( i.e. with no duration).
```
Comment (Margarita-Staneva) [inline docs/data-sources/event.md:33]: Fix spacing:

If set to `true`, creates a point-in-time event (i.e. with no duration).

File: docs/data-sources/events.md [hunk 49]
```diff
@@ -0,0 +1,33 @@
...
+
+* `events` -List of all events in Wavefront. For each event you will see a list of attributes.
+  * `name` - The name of the event in Wavefront.
+  * `id` - The ID of the event in Wavefront.
+  * `start_time`- The start time of the event in epoch milliseconds.
+  * `end_time` - The end time of the event in epoch milliseconds.
+  * `severity` - The severity category of the event.
+  * `type` - The type of the event.
+  * `details` - The description of the event.
+  * `is_ephemeral` - A Boolean flag. If set to `true`, creates a point-in-time event( i.e. with no duration).
```
Comment (Margarita-Staneva) [inline docs/data-sources/events.md:30]: Fix spaces:

If set to true, creates a point-in-time event (i.e. with no duration).

File: docs/data-sources/dashboard.md [hunk 50]
```diff
@@ -0,0 +1,157 @@
...
+* `sparkline_display_postfix` - For the single stat view, a string to append to the displayed text.
+* `sparkline_size` - For the single stat view, this determines whether the sparkline of the statistic is displayed in the chart. Valid options are `BACKGROUND`, `BOTTOM`, `NONE`.
+* `sparkline_line_color` - For the single stat view, the color of the line. Values should be in RGBA format.
+* `min` - Min value of the Y-axis. Set to null or leave blank for auto.
+* `plain_markdown_content` - The markdown content for a Markdown display, in plain text.
+* `sparkline_fill_color` - For the single stat view, the color of the background fill. Values should be in RGBA format.
+* `sparkline_value_color_map_colors` - For the single stat view, a list of colors that differing query values map to. Must contain one more element than `sparkline_value_color_map_values_v2`. Values should be in RGBA format.
+* `sparkline_value_color_map_values_v2` - For the single stat view, a list of boundaries for mapping different query values to colors.  Must contain one element less than `sparkline_value_color_map_colors`.
```
Comment (Margarita-Staneva) [inline docs/data-sources/dashboard.md:135]: There's double space between the sentences. Same applies to line 139 in this file and to the same chart settings in the dashboards.md file. This is not a big deal, but we usually use a single space after the full stop :).

File: docs/data-sources/dashboards.md [hunk 51]
```diff
@@ -0,0 +1,149 @@
...
+  * `window_size` - Width, in minutes, of the time window to use for `last` windowing.
+  * `show_hosts` - For the tabular view, whether to display sources. Default is `true`.
+  * `show_labels` - For the tabular view, whether to display labels. Default is `true`.
+  * `show_raw_values` - For the tabular view, whether to display raw values. Default is `false`.
+  * `auto_column_tags` - This setting is deprecated.
+  * `column_tags` - This setting is deprecated.
+  * `tag_mode` - For the tabular view, which mode to use to determine which point tags to display. Valid options are `all`, `top`, or `custom`.
+  * `num_tags` - For the tabular view defines how many point tags to display.
+  * `custom_tags` - For the tabular view, a list of point tags to display when using the `custom` tag display mode.
+  * `group_by_source` - For the tabular view, whether to group multi metrics into a single row by a common source. If `false`, each source is displayed in its own row.  if `true`, multiple metrics for the same host are displayed as different columns in the same row.
```
Comment (Margarita-Staneva) [inline docs/data-sources/dashboards.md:91]: For the tabular view, whether to group multi metrics into a single row by a common source. If set to `false`, each source is displayed in its own row.  If set to `true`, multiple metrics for the same host are displayed as different columns in the same row.

File: docs/data-sources/dashboard.md [hunk 52]
```diff
@@ -0,0 +1,157 @@
...
+* `window_size` - Width, in minutes, of the time window to use for `last` windowing.
+* `show_hosts` - For the tabular view, whether to display sources. Default is `true`.
+* `show_labels` - For the tabular view, whether to display labels. Default is `true`.
+* `show_raw_values` - For the tabular view, whether to display raw values. Default is `false`.
+* `auto_column_tags` - This setting is deprecated.
+* `column_tags` - This setting is deprecated.
+* `tag_mode` - For the tabular view, which mode to use to determine which point tags to display. Valid options are `all`, `top`, or `custom`.
+* `num_tags` - For the tabular view defines how many point tags to display.
+* `custom_tags` - For the tabular view, a list of point tags to display when using the `custom` tag display mode.
+* `group_by_source` - For the tabular view, whether to group multi metrics into a single row by a common source. If `false`, each source is displayed in its own row.  if `true`, multiple metrics for the same host are displayed as different columns in the same row.
```
Comment (Margarita-Staneva) [inline docs/data-sources/dashboard.md:99]: For the tabular view, whether to group multi metrics into a single row by a common source. If set to `false`, each source is displayed in its own row.  If set to `true`, multiple metrics for the same host are displayed as different columns in the same row.

**Review by Margarita-Staneva — COMMENTED** [review]
I have some comments. Please take a look.

File: docs/data-sources/dashboard.md (hunk 52 above)
Comment (dkuldeep22) [reply docs/data-sources/dashboard.md]: Thank you @Margarita-Staneva :) !!

File: docs/data-sources/event.md [hunk 53]
```diff
@@ -0,0 +1,35 @@
...
+## Attribute Reference
+
+* `name` - The name of the event in Wavefront.
+* `id` - The ID of the event in Wavefront.
+* `start_time`- The start time of the event in epoch milliseconds.
+* `end_time` - The end time of the event in epoch milliseconds.
+* `severity` - The severity category of the event.
+* `type` - The type of the event.
+* `details` - The description of the event.
+* `is_ephemeral` - A Boolean flag. If set to `true`, creates a point-in-time event(i.e. with no duration).
```
Comment (Margarita-Staneva) [inline docs/data-sources/event.md:33]: Please add space before the opening bracket:

A Boolean flag. If set to `true`, creates a point-in-time event (i.e. with no duration).

File: docs/data-sources/events.md [hunk 54]
```diff
@@ -0,0 +1,33 @@
...
+
+* `events` -List of all events in Wavefront. For each event you will see a list of attributes.
+  * `name` - The name of the event in Wavefront.
+  * `id` - The ID of the event in Wavefront.
+  * `start_time`- The start time of the event in epoch milliseconds.
+  * `end_time` - The end time of the event in epoch milliseconds.
+  * `severity` - The severity category of the event.
+  * `type` - The type of the event.
+  * `details` - The description of the event.
+  * `is_ephemeral` - A Boolean flag. If set to `true`, creates a point-in-time event(i.e. with no duration).
```
Comment (Margarita-Staneva) [inline docs/data-sources/events.md:30]: Please add space before the opening bracket:

A Boolean flag. If set to `true`, creates a point-in-time event (i.e. with no duration).

**Review by Margarita-Staneva — COMMENTED** [review]
2 minor things. Rest looks good :)

File: docs/data-sources/event.md (hunk 53 above)
Comment (dkuldeep22) [reply docs/data-sources/event.md]: Done. Thanks!

**Pull request merged**